src/test/librbd/test_librbd.cc

```diff
@@ -1948,6 +1948,7 @@ TEST_F(TestLibRBD, TestIO)
   uint64_t size = 2 << 20;  
 
   ASSERT_EQ(0, create_image(ioctx, name.c_str(), size, &order));
+  ASSERT_EQ(0, rados_conf_set(_cluster, "rbd_read_from_replica_policy", "balance"));
```

Comment:
This won't really test anything -- instead you should add a new YAML snippet here [1] that overrides the config setting and ensures the min compatible OSD release is set to Octopus.

[1] https://github.com/ceph/ceph/tree/master/qa/suites/rbd/qemu/features